### ISSUE TITLE
How to combine two trained data files : tam.traineddata and mal.traineddata 

### ISSUE THREAD

**Issue by Amricreate** [tesseract-ocr/tesstrain#396]
I am actually working on training Grantha script. What I figured out is that both Tamil and Malayalam have almost same characters and formatting as that of Grantha. 

So, I assume if both models are used simutaneously, then the combined model can be used for finetuning using my Grantha data. I am not sure whether it will work. What is your opinion? Is this the only way like finetuning each model separately and then use this command:

"tesseract image.png -l grantha+grantha1"

**Comment by Shreeshrii** [comment]
Have you had any success in developing a model for Grantha script? There are others willing to collaborate on such a project. Please respond with any progress you have made. Thanks.

**Comment by Amricreate** [comment]
@Shreeshrii 

Indeed. But it would work better on printed images but mercilessly fail on handwritten images. 

It would be grateful if I could be a part of that project.
